doc/common/Typespec-Project-Scripts.md

```diff
@@ -68,31 +68,31 @@ function Get-dotnet-EmitterPackageJsonPath() {
 
 #### Get-${Language}-EmitterAdditionalOptions (Optional)
 
-This function allows you to append additional `--option` arguments that will be passed into cadl compile.  One example of this is the `emitter-output-dir`.  For dotnet we want the location of the generated files to be `{projectDir}/src` however in other languages they will have other conventions.  This method will take in a fully qualified path to the project directory so you can construct your relative path to that as the output.
+This function allows you to append additional `--option` arguments that will be passed into typespec compile.  One example of this is the `emitter-output-dir`.  For dotnet we want the location of the generated files to be `{projectDir}/src` however in other languages they will have other conventions.  This method will take in a fully qualified path to the project directory so you can construct your relative path to that as the output.
 
 Example
 
 ```powershell
 function Get-dotnet-EmitterAdditionalOptions([string]$projectDirectory) {
-  return "--option @azure-tools/cadl-csharp.emitter-output-dir=$projectDirectory/src"
+  return "--option @azure-tools/typespec-csharp.emitter-output-dir=$projectDirectory/src"
 }
 ```
 
 ## Per project setup
 
-Each project will need to have a configuration file that will tell the scripts where to find the cadl spec.
+Each project will need to have a configuration file that will tell the scripts where to find the typespec spec.
 
-### cadl-location.yaml
+### typespec-location.yaml
```

Comment:
i think we should do `tsp-location.yaml` instead. I think it corresponds better with the reaming we've seen in our cadl files, i.e. `cadl-config.yml` is now `tspconfig.yml` etc

Reply:
Thanks @iscai-msft Good suggestion. I updated.